AGPs/Association-2020-Budget.md

```diff
@@ -0,0 +1,131 @@
+- Decentralizing all roles and responsibilities in order to transfer full governance power to ANT holders.
+
+In order to fulfill the above responsibilities for the Aragon Network, the Association needs capital to support it's team and cover it's various expenses for the Aragon Netw. 
+
+Therefore, it requests authorization to use the following funds from the Aragon treasury:
+
+- 375k DAI, for operational costs.
+- 375,000 ANT vested over the next 2 years, for the long-term incentivization of the current team, future employees and new Aragon Advisory board.
+
+### **Scope**
+
+#### **Legal**
+- Drive the decentralization process of the Aragon Association into the Aragon Network.
+- Provide legal support to Aragon teams.
+- Ensure full compliance and minimize legal risks for the continued development of the project.
+
+### **Treasury management**
+- Ensure the project has the resources it needs to continue and finish development of the network.
+- Diversify the capital base according to the project's needs.
+- Initiate and track payments to various teams, DAOs, and grant programs.
+
+#### **Events**
+
+- Design and implement a bottom-up events strategy including decentralised events and meet-ups around the world
+
+#### Grants
+- Actively look for new teams that want to work on Aragon's products and infrastructure
+- Actively look for and encourage teams to build Aragon specific applications
+- Analyze team's applications
+- Manage the grant process and help teams navigate it
+- Monitor and track milestones and teams contributions
+- Evaluate contributions and act as a point-of-contact and liaison with other Aragon Flock teams to coordinate technical evaluations of commitments
+- Oversee the decentralization of the various grants programs
+- Clearly communicate with the Community all relevant updates regarding the grants programs by maintaining an open transparency portal
+
+#### **Aragon Communications**
+- Lead in spreading awareness of Aragon and decentralized organizations
+- Manage communications and content contractors
+- Manage aragon.org, blog.aragon.org, and the [Aragon.org](http://aragon.org) email and vote alerts newsletters
+
+### **Financing**
+
+#### **Estimated operating costs for 2020**
+
+The association would like to request DAI 1.5m to cover the operating costs of 2020.
+
+The estimated operating cost for 2020 include: 
+- Payroll: 30.29%
+- External services (including legal costs, HR, communications, accounting, banking, auditing, market making and more): 46.68%
+- Travel expenses: 3.03%
+- Buffer: 20%
+
+We request the ability to both spend such funds from the Association's current treasury or transfer them to an Association specific DAO (whose members would include the Association's executive board, advisory board or team members).
+
+### **ANT packages**
+
+We would like to request 375,000 ANT, with a 3-month cliff and 2-year vesting.
```

Comment:
Add a line to justify the shorter than usual cliff and vesting. (Mainly incentivization of advisory board)